### PR TITLE
Cut a 1.0 release

### PR DESCRIPTION
**Pull request by captbaritone**
I've tested this with most of our repos that use JS. It should work as a starting place.

/review @caro5 @gdtrice @jtwb @kramretak

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,16 @@
...
\ No newline at end of file
+# eslint-config-hss
+
+Hearsay Social's base JavaScript style.
+
+## Ussage
+
+Install this package, and eslint:
+
+    npm install --save-dev eslint eslint-config-hss
```
Comment (kapadiamush) [inline README.md]: this should be wrapped in back ticks, also maybe a terminal line indicator ?

    ```
        $ npm install --save-dev eslint eslint-config-hss
    ```

**Comment by kapadiamush** [comment]
lgtm

File: README.md (hunk 1 above)
Comment (captbaritone) [inline README.md]: Indenting by four spaces triggers the code formatting as well:

https://daringfireball.net/projects/markdown/syntax#precode

**Comment by hearsaydev** [comment]
Attempting a merge on behalf of **jeldredge**. Stand clear!

File: README.md (hunk 1 above)
Comment (kapadiamush) [inline README.md]: derp I used that to get the backticks to show up

**Pull request merged**